store/src/local/config.rs

```diff
@@ -0,0 +1,12 @@
+#[derive(Clone, Copy, Debug)]
+pub struct NodeStoreConfig {
+    index: EntityIndexConfig
+}
+
+impl Default for NodeStoreConfig {
+    fn default() -> Self {
+      NodeStoreconfig {
```

Comment:
Hmm, this doesn't seem to be compiled, otherwise it would have complained since `config` is missing an upper case. I think you forgot to add it to `local/mod.rs`.

But I guess you could also just remove it since it's not used ?

Reply:
Done.